test/test_variables.py

```diff
@@ -201,11 +201,11 @@ def check_request_uri(req_uri):
 
     assert (
         wait_for_record(
-            fr'^127\.0\.0\.1 /foo/foo /foo%2Ffoo\?a=b$', 'access.log'
+            fr'^127\.0\.0\.1 /foo/foo /blah%2Fblah\?a=b$', 'access.log'
         )
         is not None
     ), 'req 8080 percent'
-    assert len(findall(fr'^::1 /bar /bar\?a=b$', 'access.log')) == 2
+    assert len(findall(fr'^::1 /bar /foo/foo\?a=b$', 'access.log')) == 1
```

Comment:
This line was necessary to check the number of records in the `access.log` for a given pattern. Since the pattern has changed, you also need to adjust the number of matches in the log. So, all is good here.